core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -597,7 +610,7 @@ protected RpcOptions createRpcOptions() {
 
    private class TxReadOnlyManyHelper extends ReadOnlyManyHelper {
       @Override
-      public ReplicableCommand copyForRemote(ReadOnlyManyCommand command, List<Object> keys, InvocationContext ctx) {
+      public ReadOnlyManyCommand<?, ?, ?> copyForRemote(ReadOnlyManyCommand command, List<Object> keys, InvocationContext ctx) {
```

Comment:
@rvansa IMO it's better to say explicitly that we don't care what the key/value types are with wildcards, instead of pretending that the command is not generic.

It's true that in this case the command is basically only used in the interceptor stack, where we don't know what the key/value types are, so it's probably better to pretend it was never generic, but doing that means more compilation warnings.